examples/trusted_information_retrieval/signatures.toml

```diff
@@ -1,4 +1,5 @@
 # database_proxy
-[[signatures.3606a86d9fa01d223dc3cb092c1aceb9e3afe0ed9799ac5d5b214b1610209f8f]]
-public_key = "examples/certs/minisign/minisign.pub"
-signature = "examples/trusted_information_retrieval/database_proxy.sign"
+[signatures]
+3606a86d9fa01d223dc3cb092c1aceb9e3afe0ed9799ac5d5b214b1610209f8f = [
+  { path = "examples/trusted_information_retrieval/database_proxy.sign" },
+]
```

Comment:
Note that this key-value format now splits the information about the hash of the code to this file, while public key and signature are in the file pointed to by path. 

I think it is worth trying to think ahead how this will be used in the future, which I imagine will be a verifiable log or a simple database that ties together hash, public key, and signature of each module. e.g. if you imagine that as a mySQL table, it would be a table with three columns, so I think it makes sense to encode the three pieces of information in the same file to begin with.

WDYT?

Can be a follow-up PR, but I'm mostly curious if you think it will evolve in that direction, or do you have something else in mind @ipetr0v

Reply:
I was thinking that it would look like this:
```toml
[signatures]

# database_proxy
3606a86d9fa01d223dc3cb092c1aceb9e3afe0ed9799ac5d5b214b1610209f8f = [
  { path = "examples/trusted_information_retrieval/database_proxy.sign" },
  { url = "https://oaksignatures.io/signature1.sign" },
  { url = "https://oaksignatures.io/signature2.sign" },
]

# some_other_module
3606a86d9fa01d223dc3cb092c1aceb9e3afe0ed9799ac5d5b214b1610209f8f = [
  { url = "https://oaksignatures.io/signature3.sign" },
]
```
So that it will just contain paths/URLs of signature files. And thus signatures will accessible just by a URL.

I'm also not sure if we will need a verifiable log for signatures (we may only need it for the Oak binary hash itself).

Reply:
But why not make the hash part of the thing that is downloaded via the URL? To avoid fetching them upfront?

Reply:
this is what I am suggesting:

```toml
signatures = [
  # database_proxy
  { path = "examples/trusted_information_retrieval/database_proxy.sign" },
  { url = "https://oaksignatures.io/signature1.sign" },
  { url = "https://oaksignatures.io/signature2.sign" },
  # some_other_module
  { url = "https://oaksignatures.io/signature3.sign" },
]
```